.github/workflows/integration-tests-against-emulator.yaml

```diff
@@ -1,42 +1,39 @@
-on:
+'on':
   push:
     branches:
-    - main
-  pull_request:
+      - 6.43.x
```

Comment:
Why are we modifying this? We would want the code pushed to `main` branch to be tested against emulator.